src/libs/ajax.js

```diff
@@ -645,8 +645,8 @@ const Methods = signal => ({
 
 
 const Jupyter = signal => ({
-  clustersList: async () => {
-    const res = await fetchLeo('api/clusters?saturnAutoCreated=true', _.mergeAll([authOpts(), appIdentifier, { signal }]))
+  clustersList: async project => {
+    const res = await fetchLeo(`api/clusters${project ? `/${project}` : ''}?saturnAutoCreated=true`, _.mergeAll([authOpts(), appIdentifier, { signal }]))
```

Comment:
Should `project` be coerced into a boolean? Should work as is, so more for readability I think.

Reply:
For the moment I'm not going to worry about that, it's a valid point though